### PR TITLE
Convert md links to xml in returns sections

### PR DESCRIPTION
**Pull request by ChrisMaddock**
More towards dotnet/docs#1931.

Couple of questions on this one...

### CODE REVIEW

File: xml/System.Transactions/TransactionInterop.xml [hunk 1]
```diff
@@ -97,7 +97,7 @@
       <Docs>
         <param name="transaction">A <see cref="T:System.Transactions.Transaction" /> instance to be marshaled.</param>
         <summary>Gets an <see cref="T:System.Transactions.IDtcTransaction" /> instance that represents a <see cref="T:System.Transactions.Transaction" />.</summary>
-        <returns>An <see cref="T:System.Transactions.IDtcTransaction" /> instance that represents a <see cref="T:System.Transactions.Transaction" />.  The <see cref="T:System.Transactions.IDtcTransaction" /> instance is compatible with the unmanaged form of [ITransaction](http://msdn.microsoft.com/library/default.asp?url=/library/cossdk/html/d40264be-f9b2-4044-aac6-84b43e5be666.asp) used by MSDTC and with the Managed form of <see cref="T:System.EnterpriseServices.ITransaction" /> used by <see cref="N:System.EnterpriseServices" />.</returns>
```
Comment (ChrisMaddock) [inline xml/System.Transactions/TransactionInterop.xml]: This link is dead, but I couldn't find a suitable replacement. Any suggestions?

File: xml/System.Windows.Media.Effects/BitmapEffect.xml [hunk 2]
```diff
@@ -164,7 +164,7 @@
       <Parameters />
       <Docs>
         <summary>Creates a handle to an [IMILBitmapEffect](http://msdn.microsoft.com/library/4bd5697d-bdde-4485-a05b-b3fea1403e16) object that is used to initialize a custom effect.</summary>
-        <returns>A handle to an [IMILBitmapEffect](http://msdn.microsoft.com/library/4bd5697d-bdde-4485-a05b-b3fea1403e16) object.</returns>
```
Comment (ChrisMaddock) [inline xml/System.Windows.Media.Effects/BitmapEffect.xml]: This MSDN link re-directs to https://docs.microsoft.com/en-gb/previous-versions/windows/desktop/api/mileffects/nn-mileffects-imilbitmapeffect, which is a dead link. Not sure if that should be fixed?

File: xml/System.Web.Services.Description/SoapProtocolImporter.xml [hunk 3]
```diff
@@ -293,7 +293,7 @@
         <param name="uriList">A space-delimited list of URIs.</param>
         <summary>Checks for the presence of "http://schemas.xmlsoap.org/soap/encoding/" in a string that represents a list of Uniform Resource Indicators (URIs).</summary>
         <returns>
-          <see langword="true" /> it the string contains [http://schemas.xmlsoap.org/soap/encoding/](http://schemas.xmlsoap.org/soap/encoding/); otherwise <see langword="false" />.</returns>
+          <see langword="true" /> it the string contains "http://schemas.xmlsoap.org/soap/encoding/"; otherwise <see langword="false" />.</returns>
```
Comment (ChrisMaddock) [inline xml/System.Web.Services.Description/SoapProtocolImporter.xml]: This isn't 'semantically' a link, so I removed it to be consistent with the summary section above. Let me know if I should put it back! 🙂

File: xml/System.Transactions/TransactionInterop.xml (hunk 1 above)
Comment (rpetrusha) [reply xml/System.Transactions/TransactionInterop.xml]: <https://docs.microsoft.com/en-us/previous-versions/windows/desktop/ms686531(v=vs.85)>

File: xml/System.Web.Services.Description/SoapProtocolImporter.xml (hunk 3 above)
Comment (rpetrusha) [reply xml/System.Web.Services.Description/SoapProtocolImporter.xml]: This still renders as a link. I think the best choice here is \<see langword="http://schemas.xmlsoap.org/soap/encoding/" />.

File: xml/System.Windows.Media.Effects/BitmapEffect.xml (hunk 2 above)
Comment (rpetrusha) [reply xml/System.Windows.Media.Effects/BitmapEffect.xml]: Thanks, @ChrisMaddock. I've reported the bad redirect.

**Comment by ChrisMaddock** [comment]
Thanks @rpetrusha, sorry it's been a while. Changes made!

**Pull request merged**